.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
+      - zuul: zuul-distro-jobs
+    requires: repo
+#    require-projects:
+#      - rpm-software-management/ci-dnf-stack
+    branches:
+      - master
+    vars:
+      dnf_master_copr: fbo/dnf-master
+    run: zuul-ci/run-tests.yaml
+    nodeset: fedora-31-vm
+
+- project:
+    check:
+      jobs:
+        - dnf-rpm-build
```

Comment:
dnf-rpm-build is defined in https://pagure.io/fedora-zuul-jobs/blob/master/f/zuul.d/dnf-jobs.yaml

Reply:
Zuul loads its config from git repos, each known git repos is scanned for CI definitions.